src/Serializer/src/Serializer/PhpSerializer.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Spiral\Serializer\Serializer;
+
+use Spiral\Serializer\SerializerInterface;
+
+final class PhpSerializer implements SerializerInterface
+{
+    public function serialize(mixed $payload): string|\Stringable
+    {
+        return serialize($payload);
+    }
+
+    public function unserialize(\Stringable|string $payload, object|string|null $type = null): mixed
+    {
+        return unserialize((string) $payload);
```

Comment:
If the `$type` argumend has been passed should we process it?
For example:
$type is class name -> we should check the unserialize result is instance of $type;
$type is object -> throw an exception about unsupported argument type OR hydrate that object

Reply:
Added exception.